src/Ansatz.jl

```diff
@@ -437,13 +437,13 @@ function simple_update_1site!(ψ::AbstractAnsatz, gate)
 end
 
 function simple_update_2site!(
-    ::MixedCanonical, ψ::AbstractAnsatz, gate; threshold=nothing, maxdim=nothing, normalize=false
+    ::MixedCanonical, ψ::AbstractAnsatz, gate; threshold=nothing, maxdim=nothing, normalize=false, kwargs...
```

Comment:
You can replace the other kwargs in here right? Because they are not directly used but just passed.

```suggestion
    ::MixedCanonical, ψ::AbstractAnsatz, gate; kwargs...
```

Reply:
True! I will add that.